server/package.json

```diff
@@ -12,20 +12,23 @@
     "url": "https://github.com/chitsaou/vscode-coffeescript-support"
   },
   "dependencies": {
-    "coffeescript": "^2.1.0",
-    "vscode-languageserver": "^3.4.2"
+    "coffeescript-symbols": "file:../coffeescript-symbols",
+    "coffeescript-symbol-index": "file:../coffeescript-symbol-index",
```

Comment:
Problem: these packages won't be `npm install`'d during `npm run compile`.

Reply:
Even if unit test passes, the built vsix won't run due to lack of these packages.

Publishing them to NPM could solve the problem, but that's overkill.